src/sentry/integrations/jira/integration.py

```diff
@@ -303,7 +303,7 @@ def get_issue(self, issue_id, **kwargs):
         return {
             "key": issue_id,
             "title": issue["fields"]["summary"],
-            "description": issue["fields"]["description"],
+            "description": issue["fields"].get("description", None),
```

Comment:
the default should be `None` if the description isn't there, so we shouldn't need it here